app/poros/photo.rb

```diff
@@ -0,0 +1,9 @@
+class Photo
+  attr_reader :id,
+              :url
+
+  def initialize(photo_data)
+    @id = nil
+    @url = photo_data[:results][0][:urls][:raw]
+  end
+end
```

Comment:
Sorry folks if you keep searching the same city and the same photo populates over and over